test/behavior/generated-parser-behavior.spec.js

```diff
@@ -2677,13 +2677,13 @@ describe("generated parser behavior", () => {
         });
 
         it("reports expectations correctly with multiple alternatives", () => {
-          const parser = peg.generate("start = 'a' / 'b' / 'c'", options);
+          const parser = peg.generate("start = 'aa' / 'bb' / 'cc'", options);
 
           expect(parser).to.failToParse("d", {
             expected: [
-              { type: "literal", text: "a", ignoreCase: false },
-              { type: "literal", text: "b", ignoreCase: false },
-              { type: "literal", text: "c", ignoreCase: false },
+              { type: "literal", text: "aa", ignoreCase: false },
+              { type: "literal", text: "bb", ignoreCase: false },
+              { type: "literal", text: "cc", ignoreCase: false },
             ],
```

Comment:
Strictly speaking, we should give parser a chance to not fail by parsing `dd` here and below :)